packages/f-checkout/src/constants.js

```diff
@@ -13,3 +13,14 @@ export const TENANT_MAP = {
     'it-IT': 'it',
     'nb-NO': 'no'
 };
+
+export const FULFILLMENT_TIMES = [
+    {
+        from: 'beginning',
+        label: {
+            text: 'time 1'
+        },
+        selected: false,
+        to: 'end'
+    }
+];
```

Comment:
I don't think this should live here, to be honest. This file has constants for the whole component, whereas these fulfillment times are for testing purposes only. I'd move these to the test file.

Reply:
cc @ashleynolan, since I now realise he suggested the opposite thing 😬.